packages/visualizations-react/package.json

```diff
@@ -14,14 +14,20 @@
         "TypeScript",
         "React"
     ],
-    "main": "dist/index.umd.js",
+    "main": "dist/index.es.js",
     "module": "dist/index.es.js",
-    "unpkg": "dist/index.umd.js",
     "typings": "dist/src/index.d.ts",
     "files": [
         "dist",
         "src"
     ],
+    "exports": {
```

Comment:
I get this error in platform: 
> Module not found: Error: Package path ./dist/index.css is not exported from package /usr/src/app/ods/react/node_modules/@opendatasoft/visualizations-react (see exports field in /usr/src/app/ods/react/node_modules/@opendatasoft/visualizations-react/package.json)

Adding `"./dist/": "./dist/"` seems to fix the warning indeed